modules/dnn/src/net_impl_fuse.cpp

```diff
@@ -209,8 +209,9 @@ void Net::Impl::fuseLayers(const std::vector<LayerPin>& blobsToKeep_)
 
                 if (!nextData->params.has("operation") || toLowerCase(nextData->params.get<String>("operation")) != "add")
                 {
+                    String operation_text = toLowerCase(nextData->params.get<String>("operation", "sum"));
```

Comment:
> "sum"

Why is `sum`?
`<empty>` should be more appropriate.

Reply:
Please take a look: https://github.com/opencv/opencv/blob/4.x/modules/dnn/src/layers/eltwise_layer.cpp#L105